.buildkite/pipeline.yml

```diff
@@ -1,5 +1,38 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
 
+env:
+  SETUP_GOLANG_VERSION: '1.17.6'
```

Comment:
Should we use the same mechanism as @sharbuz to set the GO_VERSION from the file in the pre-command hook ?

Reply:
ok > updated